src/main/java/uk/ac/ebi/ena/readtools/cram/Cram2Fastq.java

```diff
@@ -183,57 +181,49 @@ public Dumper(InputStream cramIS, CRAMReferenceSource referenceSource, int nofSt
 		protected abstract void containerHasBeenRead() throws IOException;
 
 		protected void doRun() throws IOException {
-			cramHeader = CramIO.readCramHeader(cramIS);			
+			cramHeader = CramIO.readCramHeader(cramIS);
 
 			reader = newReader();
```

Comment:
Confusing use of variable name 'reader'. Best just to declare here:
MultiFastqOutputter fastqOutputter = createMultiFastqOutputter();

Note that newReader() has been renamed to createMultiFastqOutputter()